organelle_morphology/project.py

```diff
@@ -144,6 +145,69 @@ def add_source(
 
         self._sources[source] = source_obj
 
+    def _create_plotly_meshes(self, ids: str = "*", show_morphology: bool = False):
+        # filter organelles
+
+        return_ids = False
+        orgs = self.organelles(ids, return_ids)
+
+        meshes_to_draw = []
+
+        for org in orgs:
+            verts = org.mesh.vertices
+            faces = org.mesh.faces
+
+            # prepare data for plotly
+            vertsT = np.transpose(verts)
+            facesT = np.transpose(faces)
+
+            if show_morphology:
+                curvature_vertices = org.morphology_map
+                intensity = curvature_vertices
+                colorscale = "Viridis"
+                opacity = 1
+            else:
+                intensity = None
+                colorscale = None
+                opacity = 1
+
+            go_mesh = go.Mesh3d(
+                x=vertsT[0],
+                y=vertsT[1],
+                z=vertsT[2],
+                i=facesT[0],
+                j=facesT[1],
+                k=facesT[2],
+                name=org.id,
+                opacity=opacity,
+                # note: opacity below 1 seems to be an ongoing issue with plotly in 3d.
+                # shapes might not be drawn in the correct order and overlap wierdly when moving the camera,
+                intensity=intensity,
+                colorscale=colorscale,
+                showscale=False,
+            )
+            meshes_to_draw.append(go_mesh)
+
+        return meshes_to_draw
+
+    def show(self, ids: str = "*", show_morphology: bool = False):
+        meshes_to_draw = self._create_plotly_meshes(ids, show_morphology)
```

Comment:
Is the construction of the Plotly meshes expensive? If so, they should probably also be stored on the object. If not, I am not sure the additional `_create_plotly_meshes` method is required or we could just created them in the below loop.